python/cudf/cudf/utils/applyutils.py

```diff
@@ -148,7 +161,7 @@ def run(self, df, **launch_params):
         # Prepare output frame
         outdf = df.copy()
         for k in sorted(self.outcols):
-            outdf[k] = outputs[k]
+            outdf[k] = Series(outputs[k], nan_as_null=False)
```

Comment:
This is different nan/null behavior than before, AFAICT. Any specific reason?

Reply:
I figured if a user explicitly passes in data with `NaN` values from my perspective they wouldn't expect to get `null` values back. There's still the pessimistic null handling for handling actual nulls.

Reply:
Good change.